osmoutils/coins/coin_math.go

```diff
@@ -0,0 +1,114 @@
+package coins
+
+import (
```

Comment:
Note to reviewer:  the helpers in these two files have been merged in https://github.com/osmosis-labs/osmosis/pull/6427

Since this PR is made off of https://github.com/osmosis-labs/osmosis/pull/6366, we must wait for #6366 to be merge in first before removing this from the diff